images/virtualization-artifact/pkg/controller/vmip/vmip_webhook.go

```diff
@@ -71,14 +78,22 @@ func (v *Validator) ValidateCreate(ctx context.Context, obj runtime.Object) (adm
 					allocatedLease.Spec.VirtualMachineIPAddressRef.Name != vmip.Name) {
 				return nil, fmt.Errorf("VirtualMachineIPAddress cannot be created: the IP address %s has already been allocated by VirtualMachineIPAddress/%s in ns/%s", ip, allocatedLease.Spec.VirtualMachineIPAddressRef.Name, allocatedLease.Spec.VirtualMachineIPAddressRef.Namespace)
 			}
+
+			err = v.ipService.IsAvailableAddress(vmip.Spec.StaticIP, allocatedIPs)
+			if err != nil {
+				if errors.Is(err, service.ErrIPAddressOutOfRange) {
+					warnings = append(warnings, fmt.Sprintf("The requested address %s is out of the valid range",
+						vmip.Spec.StaticIP))
+				}
```

Comment:
An error was lost. If it's not a warning, log it

Reply:
It is a warning.